airflow/operators/dagrun_operator.py

```diff
@@ -144,6 +144,9 @@ def execute(self, context: Dict):
                 dag = dag_bag.get_dag(self.trigger_dag_id)
 
                 dag.clear(start_date=self.execution_date, end_date=self.execution_date)
+
+                dag_run = DagRun.find(dag_id=dag.dag_id, run_id=run_id)[0]
```

Comment:
I am thinking if `dag_run = dag.get_last_dagrun()` would work?

Reply:
@turbaszek fixed :)

Reply:
I'm sorry but I'm not sure if my suggestion was good. We retrieve the dag run when one already exist. My suggestion would _probably_ work in case of triggering new one. I think your original suggestion was more ok

Reply:
@turbaszek you would only be in the if statement if there is a dag_run to get.